### PR TITLE
Update unsupported.js

### PR DESCRIPTION
**Pull request by audinue**
This small pull request simply changes the warning message so that it will make more sense in the future :smile: 

**Before**
```
> npm -v
npm WARN npm npm does not support Node.js v10.11.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
6.4.1
```

The current version of node is v10.11.0 but should I *upgrade* to the *newer* 4, 6, 7, 8, 9? :confused: 

**After**
```
> npm -v
npm WARN npm npm does not support Node.js v10.11.0
npm WARN npm You should probably install the supported version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
6.4.1
```

### CODE REVIEW

**Comment by ljharb** [comment]
The logic in this file shouldn’t error for node v10.11, and node 10 and 11 are both listed in the list.

**Review by zkat — APPROVED** [review]
Yeah this makes more sense :)

File: lib/utils/unsupported.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ exports.checkForUnsupportedNode = function () {
     var log = require('npmlog')
     var supportedMajors = supportedNode.map(function (n) { return n.ver }).join(', ')
     log.warn('npm', 'npm does not support Node.js ' + process.version)
-    log.warn('npm', 'You should probably upgrade to a newer version of node as we')
+    log.warn('npm', 'You should probably install the supported version of node as we')
```
Comment (ljharb) [inline lib/utils/unsupported.js]:
```suggestion
    log.warn('npm', 'You should probably install a supported version of node as we')
```

Comment (audinue) [reply lib/utils/unsupported.js]: Thanks :smile:

**Pull request closed without merging**